app-text/zathura/zathura-0.5.11.ebuild

```diff
@@ -1,44 +1,43 @@
-# Copyright 1999-2024 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
 inherit meson virtualx xdg
 
-DESCRIPTION="A highly customizable and functional document viewer"
+DESCRIPTION="Highly customizable & functional document viewer"
 HOMEPAGE="https://pwmt.org/projects/zathura/"
 
 if [[ ${PV} == *9999 ]]; then
 	inherit git-r3
 	EGIT_REPO_URI="https://git.pwmt.org/pwmt/${PN}.git"
 	EGIT_BRANCH="develop"
 else
-	SRC_URI="
-		https://github.com/pwmt/zathura/archive/${PV}.tar.gz -> ${P}.tar.gz
-	"
+	SRC_URI="https://github.com/pwmt/zathura/archive/${PV}.tar.gz -> ${P}.tar.gz"
 	KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86 ~amd64-linux ~x86-linux"
 fi
 
 LICENSE="ZLIB"
-SLOT="0/5.6"
-IUSE="man seccomp synctex test"
-
+SLOT="0/6.7" # plugin versions api.abi (see meson.build)
+IUSE="+man seccomp synctex test"
 RESTRICT="!test? ( test )"
 
 RDEPEND="
 	dev-libs/json-glib
-	man? ( dev-python/sphinx )
-	seccomp? ( sys-libs/libseccomp )
-	synctex? ( app-text/texlive-core )
 	sys-apps/file
 	x11-libs/cairo
+	x11-libs/pango
 	>=dev-db/sqlite-3.6.23:3
-	>=dev-libs/girara-0.4.3:=
+	>=dev-libs/girara-0.4.5:=
```

Comment:
It seems girara needs an adaptation too (adding X useflag) ... I get an error in a global environnement without X: \*/* -X wayland: 

```
/usr/lib/gcc/x86_64-pc-linux-gnu/16/../../../../x86_64-pc-linux-gnu/bin/ld: /usr/lib64/libgirara-gtk3.so: undefined reference to `gtk_plug_new
'
collect2: error: ld returned 1 exit status
```
See: 
https://github.com/search?q=repo%3Apwmt%2Fgirara%20GDK_WINDO&type=code